### PR TITLE
0.5.2 changelog and release

### PR DESCRIPTION
**Pull request by neiljp**
This is a draft of commits with release notes/changes to prepare for the 0.5.2 release, which we can hopefully release soon.

### CODE REVIEW

File: zulipterminal/version.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ZT_VERSION = '0.5.1+git'
+ZT_VERSION = '0.5.2'
 
 SUPPORTED_SERVER_VERSIONS = [
     ('2.1', None),
```
Comment (preetmishra) [inline zulipterminal/version.py:4]: Let's remove the comment from the next line.

Comment (neiljp) [reply zulipterminal/version.py:4]: Oh yes, I agree - though that's not shown in this review context...

Comment (neiljp) [reply zulipterminal/version.py:4]: That said, there are features of v3.0 that we don't yet support. Perhaps #731 could include a table for version-specific features.

**Comment by neiljp** [comment]
This was used as the basis for the 0.5.2 release notes, so closing this now :tada:

**Pull request closed without merging**